src/libostree/ostree-sysroot-deploy.c

```diff
@@ -2967,9 +2967,20 @@ sysroot_finalize_selinux_policy (int deployment_dfd, GError **error)
     "semodule", "--help"
   };
   static const gsize SEMODULE_HELP_ARGC = sizeof (SEMODULE_HELP_ARGV) / sizeof (*SEMODULE_HELP_ARGV);
-  if (!run_in_deployment (deployment_dfd, SEMODULE_HELP_ARGV,
-                          SEMODULE_HELP_ARGC, &exit_status, &stdout, error))
-    return FALSE;
+  {
+    g_autoptr(GError) local_error = NULL;
+    if (!run_in_deployment (deployment_dfd, SEMODULE_HELP_ARGV,
+                            SEMODULE_HELP_ARGC, &exit_status, &stdout, &local_error))
+      {
+        if (g_error_matches (local_error, G_SPAWN_ERROR, G_SPAWN_ERROR_NOENT))
+          {
+            ot_journal_print (LOG_INFO, "No bwrap available");
```

Comment:
A better log message here would be nice. Maybe

```suggestion
            ot_journal_print (LOG_INFO, "Skipping `semodule --rebuild-if-modules-changed`: no bwrap available");
```

?